src/status_im/extensions/capacities/components.cljs

```diff
@@ -1,16 +1,19 @@
 (ns status-im.extensions.capacities.components
-  (:require [status-im.ui.components.react :as react]
-            [status-im.chat.commands.impl.transactions :as transactions]
-            [status-im.ui.components.button.view :as button]
-            [re-frame.core :as re-frame]
-            [status-im.utils.platform :as platform]
-            [status-im.ui.components.list.views :as list]
-            [status-im.ui.components.checkbox.view :as checkbox]
-            [status-im.ui.components.colors :as colors]
-            [status-im.ui.components.icons.vector-icons :as icons]
-            [status-im.ui.components.tooltip.views :as tooltip]
-            [status-im.ui.components.text-input.styles :as styles]
-            [status-im.extensions.capacities.map :as map-component]))
+  (:require
+   [reagent.core :as reagent]
```

Comment:
We usually use previous format

Reply:
I don't know why these spaces was inserted, but I Solved now 👍